rwslib/tests/test_rwscmd.py

```diff
@@ -13,21 +15,18 @@ def setUp(self):
 
     @httpretty.activate
     def test_version(self):
-
         httpretty.register_uri(
             httpretty.GET, "https://innovate.mdsol.com/RaveWebServices/version",
             status=200,
             body='1.0.0')
 
-        result = self.runner.invoke(rwscmd.rws, ['https://innovate.mdsol.com','version'],
+        result = self.runner.invoke(rwscmd.rws, ['https://innovate.mdsol.com', 'version'],
```

Comment:
OK, I set it so the username/password default to the empty string, and that if nothing is passed an unauth RWSConnection is configured.  Please check that you are ok with the solution